bankaccount_test.go

```diff
@@ -29,6 +29,16 @@ func TestBankAccount_UnmarshalJSON(t *testing.T) {
 	}
 }
 
+func TestBankAccountListParams_AppendTo(t *testing.T) {
+	{
+		params := &BankAccountListParams{}
+		body := &form.Values{}
+		form.AppendTo(body, params)
+		t.Logf("body = %+v", body)
+		assert.Equal(t, []string{"bank_account"}, body.Get("object"))
+	}
```

Comment:
should you be testing Account and Customer like you do for cards?

Reply:
Yeah, I skipped it because the same conditional branch just adds `object` regardless of the circumstances, but I guess it can't hurt to be explicit. Broke it into two test cases like cards.